manim/mobject/three_dimensions.py

```diff
@@ -153,6 +156,81 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
             face.set_fill(colors[c_index], opacity=opacity)
         return self
 
+    def set_fill_by_value(self, axes: "Mobject", colors: Union[Iterable[Color], Color]):
+        """Sets the color of each face of a parametric surface to a color relative to it's z-value
```

Comment:
```suggestion
        """Sets the color of each face of a parametric surface to a color relatively to its z-value.
```

Reply:
I agree that "it's" should be changed to "its", I get that wrong from time to time.  But "relative" is more grammatically correct than "relatively" in this case.  I will update via a commit along with the update further explaining `axes` below.  Does that work?